packages/react-instantsearch-core/src/connectors/connectSmartSort.ts

```diff
@@ -0,0 +1,77 @@
+import createConnector, { ConnectedProps } from '../core/createConnector';
+import {
+  refineValue,
+  getCurrentRefinementValue,
+  getResults,
+} from '../core/indexUtils';
+
+function getId() {
+  return 'relevancyStrictness';
+}
+
+function getCurrentRefinement(
+  props: ConnectedProps<any>,
+  searchState,
+  context
+) {
+  const id = getId();
+  const currentRefinement = getCurrentRefinementValue(
+    props,
+    searchState,
+    context,
+    id
+  );
+
+  return currentRefinement;
+}
+
+export default createConnector({
+  displayName: 'AlgoliaSmartSort',
+
+  getProvidedProps(props, searchState, searchResults) {
```

Comment:
```suggestion
  getProvidedProps(props, _searchState, searchResults) {
```

I think this will remove the ts error